### ISSUE TITLE
Icons missing from CHM help file

### ISSUE THREAD

**Issue by ramayasket** [EWSoftware/SHFB#629]
I'm building a CHM file based on DLL and XML file. A help file is built, but its icons are missing.

I expect them to be embedded within CHM so I can give it out as a single file. Instead, they are referenced to /icons directory, and the CHM file don't see them. That is, so classes, properties, methods are missing respective icons.

![image](https://user-images.githubusercontent.com/31845918/43567259-de0ead7e-9639-11e8-85e2-6b7479926cd5.png)

I also build web help and Help Viewer help, all these have correct icons

**Comment by EWSoftware** [comment]
The help files are building as expected.  See the ones included with SHFB itself.  The icons appear in those.  Are you using a custom presentation style?

**Comment by ramayasket** [comment]
These are the icons included with SHFB which are missing. Here is how Web version looks:

![image](https://user-images.githubusercontent.com/31845918/43613450-0bf33312-96b8-11e8-93a2-22ec98f84b4e.png)

Why CHM file doesn't look the same?

**Comment by ramayasket** [comment]
Now, I'm not messing with presentation styles at all.

**Comment by EWSoftware** [comment]
Are you opening the help file from your local hard drive or from a network share?  If from a network share, it's perhaps blocking the content.  If they're missing after the help file is downloaded from somewhere, you may need to unblock the file before the content will show.

The issue is probably with the help components on the system.  You could try reinstalling the HTML Help Workshop to see if that fixes anything.  You could also try building on another system to see if it works there.

**Comment by ramayasket** [comment]
1) No I'm not involving networks in any way, and the file isn't blocked.
2) Interestingly after I've added an image onto one of the pages, the icons suddenly appeared in a mysterious way.

Added image:

![image](https://user-images.githubusercontent.com/31845918/43667647-9680e208-9781-11e8-9619-09a9aa29bf88.png)

And the returned icons:

![image](https://user-images.githubusercontent.com/31845918/43667681-b4528e80-9781-11e8-90da-9ba03bea32ce.png)

I also realized that the icons were indeed embedded into the CHM, as the file size hasn't changed, and the file opens OK while moved to other directory.

I don't know how, but the issue seems to be resolved. Probably adding an image has turned some flag on or off which I couldn't find myself.
